docs/logs/fields.md

```diff
@@ -4,4 +4,68 @@ description: Sematext Logs App looks for special fields in logs, namely os.host,
+
+The structure of your data is dictated by the data shipped to Sematext Logs as log events. There are two places where you can easily see your data structure - the fields and filters section of your Logs table and the [Field Editor](/logs/fields/#field-editor):
+
+<img src="../../images/logs/logs_structure_fields_and_filters.png" alt="Logs Fields and Filters">
+
+<img src="../../images/logs/logs_structure_field_editor.png " alt="Logs Field Editor">
+
+---
+**Note:**
+The structure shown by the fields and filters section and the field editor may differ. The data structure shown by the fields and filters always covers the **whole data** - old and new. The data structure shown by the Field Editor is always the **current** one, the one that will be applied to all your current and future events.
+--- 
+
+### Field Types
+
+Each field has a field type associated with it and will include:
+
+ * [Integer/Long](/logs/field-types/#integerlong) - numerical data without floating points
+ * [Float/Double](/logs/field-types/#floatdouble) - numerical, floating point data
+ * [Boolean](/logs/field-types/#boolean) - Boolean field
+ * [Keyword/Not analyzed string](/logs/field-types/#not-analyzed-string) - not analyzed text
+ * [Text/Analyzed string](/logs/field-types/#analyzed-string) - analyzed, full text searchable data
+ * [Date](/logs/field-types/#date) - date based data
+ * [Geo](/logs/field-types/#geo) - field type dedicated for spatial data
+ * Object - nested structure holding structured data
+
+### Modifying Fields
+
+You can [add](/logs/fields/#adding-fields), [remove](/logs/fields/#removing-fields) or [edit](/logs/fields/#editing-fields) existing fields by using the [field editor] accessible via a Logs App settings or by using the [Templates and Mappings](/logs/mappings-templates) APIs directly. Keep in mind that the modification is applied only to the data that is shipped after the adjustments, so you may need to re-index the data in case you want the old data to be adjusted. 
+
+## Field Editor
+
+The Field Editor functionality allows adding, editing, and removing fields present in your data structure definition.
+
+<img src="../../images/logs/logs_field_editor_add.png " alt="Logs Field Editor - Adding Fields">
+
+### Adding Fields
+
+Field Editor allows adding new field by providing its name and [type](/logs/field-types/).   
+
+### Editing Fields
+
+Field Editor allows modifying a field type by changing its type. 
+
+<img src="../../images/logs/logs_field_editor_edit.png " alt="Logs Field Editor - Editing Fields">
+
+The changes done to the field are only applied to the data shipped after the change was done. Once the changes are applied the Field Editor will give you an option to re-index your old data - [learn more](/logs/fields/#re-indexing-data).
+
+### Removing Fields
+
+Field editor allows removing fields that are no longer used or needed. You can mark multiple fields for deletion and apply the changes once everything is ready. 
```

Comment:
editor => Editor

> no longer used or needed

no longer used, as in "no longer present in your logs"?

> or needed

So if I remove a field via FE because I no longer need it, but keep shipping logs with it, this field will not reappear anywhere in SC?  Aha, I see it answered below.

Reply:
Adjusted and to answer your question - I added the section on purpose, because we had a few questions around it and we have that ticket to fake full field deletion :) For now it will just reappear.